tests/external_botocore/_mock_external_bedrock_server.py

```diff
@@ -3818,6 +3818,2080 @@
             "inputTextTokenCount": 6,
         },
     ],
+    "cohere.embed-english-v3::This is an embedding test.": [
```

Comment:
Are these two cohere.embed models different in terms of parsing? If not, then I would just follow the existing precedent here of only testing one of them.